pipelines/skylab/optimus/scripts/docker_versions.tsv

```diff
@@ -0,0 +1 @@
+quay.io/humancellatlas/hca_preprocessing:1.0
```

Comment:
Has this script not been run yet? I guess I was expecting to see the 2.0 version in this tsv as well